bids/modeling/tests/test_transformations.py

```diff
@@ -316,8 +316,8 @@ def test_threshold(collection):
 
 
 def test_assign(collection):
-    transform.Assign(collection, 'parametric gain', target='RT',
-                     target_attr='onset', output='test1')
+    transform.Assign(collection, ['parametric gain'], target=['RT'],
```

Comment:
Do inputs now have to be lists? or do they get converted?

Reply:
Do you have a test to add which covers your original use case?

Reply:
inputs do not have to be lists, but if one of the kwargs is a list (and sync_kwargs is true), then all the other kwargs need to be a list. The non-list use case is seen below (line 328):
```
 transform.Assign(collection, 'RT', target='parametric gain',
                     input_attr='onset', target_attr='amplitude',
                     output='test2')
```

Reply:
@jdkent Do you have time to add a test that would have failed before?